src/services/settings-service/settings-service.ts

```diff
@@ -36,6 +44,14 @@ export class SettingsService {
         return this.theme.asObservable();
     }
 
+    setFundsSwitchStatus(val): void {
+        this.fundsSwitch.next(val);
```

Comment:
As commented earlier it might make sense to add a call to `saveFundsSwitchSettings` here.